Page/NavigatorPage.robot

```diff
@@ -1,7 +1,15 @@
+*** Settings ***
+Library           Selenium2Library
+
 *** Variables ***
-${TOP_LOGO_IMAGE_LOCATOR}    ${EMPTY}
+${TOP_LOGO_IMAGE_LOCATOR}    //img[@src='images/logo.gif']
 
 *** Keywords ***
 Is Top Logo Image Visible
+    Wait Until Element Is Visible    ${TOP_LOGO_IMAGE_LOCATOR}    1000
```

Comment:
"Is" should return boolean true or false. So, use 
${status} | Run Keyword And Return Status |  Wait Until Element Is Visible | ${TOP_LOGO_IMAGE_LOCATOR} | 1000
[Return] | ${status}